### ISSUE TITLE
Final parsing error

### ISSUE THREAD

**Issue by nikos-kekatos** [JuliaReach/SX.jl#19]
In the platooning [benchmark](https://github.com/JuliaReach/ReachabilityBenchmarks/blob/master/models/ARCH2018/Platooning/platoonComp_flat_revised.xml), the parsing induces the following error 

``invariant -9 <= aL <= 1 in location 1 not understood``

**Comment by mforets** [comment]
At least the parsing string works as expected:

```julia
julia> SX.parse_sxmath("-1 <= x <= 1")
1-element Array{Expr,1}:
 :(-1 <= x <= 1)
```

**Comment by mforets** [comment]
> Just a guess: You need to split double inequalities a <= x <= b into a <= x & x <= b.

Yes, i think your guess is correct. This type of expression is a `:comparison` and it was not been taken into account for the transformation of expressions to sets.

```julia
julia> using SX

julia> expr = SX.parse_sxmath("-9 <= aL <= 9")
1-element Array{Expr,1}:
 :(-9 <= aL <= 9)

julia> expr[1].head
:comparison

julia> expr[1].args
5-element Array{Any,1}:
 -9
   :(<=)
   :aL
   :(<=)
  9
```

It is easy to add a new case, checking that the args 2 and 5 are `<=` and then transform it into two halfspaces.

**Comment by mforets** [comment]
Another model with uses this conjunction notation is the [linear switching SX model](https://github.com/JuliaReach/ReachabilityBenchmarks/pull/156).
